/**: # wildcard for the node name
  ros__parameters:
    thrust_min: 0.0
    thrust_max: 100.0
    body_rate_max: 20.0
    mass: 2.6
    damping: 0.0
    t_final: 10.0
    timestep_min: 0.004
    continuous: true
    generation_update_period: 0.02
    open_loop_threshold_time: 0.1
    lookahead_factor: 1
    gravity:
      x: 0.0
      y: 0.0
      z: -9.81

